src/example_ws_usage.ts

```diff
@@ -3,12 +3,15 @@ import { PythConnection } from './PythConnection'
 import { getPythClusterApiUrl, getPythProgramKeyForCluster, PythCluster } from './cluster'
 import { PriceStatus } from '.'
 
-const SOLANA_CLUSTER_NAME: PythCluster = 'mainnet-beta'
+const SOLANA_CLUSTER_NAME: PythCluster = 'devnet'
```

Comment:
drive-by: this example fails when pointed to mainnet because the default solana mainnet RPC endpoint no longer accepts program subscribe calls.